shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -28,7 +28,7 @@
     <ul class="menubar">
       {{#each items}}
         {{#unless onlyPopup}}
-          <li class="{{name}}">
+          <li {{inlineStyle}} class="{{name}}">
```

Comment:
Since we require IE10, and Jake Weisz says requiring IE11 is the same as
requiring IE10, then I propose we declare that we require IE11, and then I
can avoid bothering with vendor prefixing, which is good not just because
it makes my life easier but also because it saves us from work down the
road.

Semi-sorry if I sound like I'm being lazy, but I'm extremely happy whenever
I can shrink the testing matrix of things I write. And a line of code (in
this case CSS) never-written is a line tested.